### PR TITLE
DataQuery: added data query aggregate endpoint

### LINKED ISSUE #759: DataQuery Aggregate endpoint
- [ ] Add endpoint to allow fetch of data aggregate:
input parameters: table, measures (with aggregate function), dimensions, filter, offset

### PR DESCRIPTION
**Pull request by SteRiccio**
Resolves #759

### CODE REVIEW

File: server/modules/surveyRdb/api/surveyRdbApi.js [hunk 1]
```diff
@@ -41,17 +52,63 @@ export const init = (app) => {
...
+
+  app.post(
+    '/surveyRdb/:surveyId/:nodeDefUuidTable/export-agg',
+    requireRecordListViewPermission,
+    async (req, res, next) => {
+      try {
+        const params = {
+          ..._getParamsQueryAgg(req),
+          streamOutput: res,
+        }
```
Comment (minotogna) [inline server/modules/surveyRdb/api/surveyRdbApi.js:105]: this can go in one line

File: server/modules/surveyRdb/api/surveyRdbApi.js [hunk 2]
```diff
@@ -41,17 +52,63 @@ export const init = (app) => {
...
+        Response.setContentTypeFile(res, 'data-aggregate.csv', null, Response.contentTypes.csv)
```
Comment (minotogna) [inline server/modules/surveyRdb/api/surveyRdbApi.js:106]: should we call the files (both raw and aggregate) data-export-${DATE}.csv ?

File: server/modules/surveyRdb/api/surveyRdbApi.js (hunk 1 above)
Comment (SteRiccio) [reply server/modules/surveyRdb/api/surveyRdbApi.js:105]: sure

File: server/modules/surveyRdb/api/surveyRdbApi.js (hunk 2 above)
Comment (SteRiccio) [reply server/modules/surveyRdb/api/surveyRdbApi.js:106]: ok

**Pull request merged**